demos/chat-client/c++/process_assets.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i python3 -p "python3.withPackages([])"
+
+# Python script from https://github.com/rjzak/web_wordpuzzle
+
+ASSETS = ( ("jquery", open("assets/jquery-3.6.0.min.js", "rb").read()), ("chat", open("assets/chat.html", "rb").read()), )
```

Comment:
Why add another development dependency for a simple script like this? Can't we use C++ functionality to take care of this directly at compilation time?
Could this be just a simple C++ program, which is run at build time?